app/MMB/src/scenes/travelDocument/form/TravelDocumentForm.js

```diff
@@ -0,0 +1,49 @@
+// @flow strict
+
+import * as React from 'react';
+import { View } from 'react-native';
+import { Translation } from '@kiwicom/mobile-localization';
+import { StyleSheet, TextIcon, Color } from '@kiwicom/mobile-shared';
+
+import TitleTranslation from '../../../components/TitleTranslation';
+import IdInput from './IdInput';
+import ExpiryDatePicker from './ExpiryDatePicker';
+
+type Props = {|
+  +fullName: string,
+  +title: string,
+|};
+
+export default function TravelDocumentForm(props: Props) {
+  return (
+    <React.Fragment>
+      <View style={styles.row}>
+        <TextIcon code="w" style={styles.icon} />
+        <TitleTranslation title={props.title} />
```

Comment:
I think this could be improved further, as you don't know that all languages will write `Mr. Bla`; it might very well be `Bla, Mr.`. So I think we should do: 
'title_translation_mr': 'Mr. __name__', and then use
```
<Translation id="title_translation_mr" values={{name: 'John Smith'}} />
```

Reply:
That might be true, but I would say that is the scope of another PR 😄

Reply:
Absolutely

Reply:
I will create an issue for it